gen2-facemesh/requirements.txt

```diff
@@ -1,3 +1,3 @@
 opencv-python
 numpy
-depthai==2.16.0.0
\ No newline at end of file
+depthai-sdk==1.9.6
```

Comment:
@Erol444 Hi - I'd like to test this PR, but I can't find the tag for `depthai-sdk==1.9.6` (I can see `1.9.5` in the `depthai` repo). Can you please help me locate the `v1.9.6` version of the `depthai-sdk`?

Reply:
Hi @mhilden , should be actually 1.10.1, let me try it out and merge it if it works

Reply:
Nope, we'll need to wait for the new release of depthai-sdk.. facemesh NN support is currently in `develop`

Reply:
Okay, @Erol444; thanks for the help. I'll wait to test this until the lastest `depthai-sdk` is out and hopefully by then I can test it on our new OAK-D SR PoE.

Reply:
doesn't work atm, I'm just fixing it up..

Reply:
@mhilden  this should now fix it: https://github.com/luxonis/depthai/commit/5863140708b25ceac03fac1662f685bb59fef479